core/src/main/java/org/frankframework/mongodb/StrictJsonDocumentWriter.java

```diff
@@ -1,5 +1,5 @@
 /*
-   Copyright 2021, 2023 WeAreFrank!
+   Copyright 2021, 2023-2024 WeAreFrank!
```

Comment:
Dit triggered me dat er nog wel een paar bestanden zijn waarin je het copyright mag updaten. 😄

Reply:
Oke, maar die hebben we dan gemist in het origineel. Ik zal die in master effe meenemen met een andere PR. Oke?

Reply:
Is prima, wat mij betreft!

Reply:
Oke top!